GTViewController/Classes/Nav Classes/GTFollowupViewController.m

```diff
@@ -101,6 +104,11 @@ - (void)setFollowupView:(GTFollowupModalView *)followupModalView andThankYouView
     return;
 }
 
+- (void)setPackageCode:(NSString *)packageCode andLanguageCode:(NSString *)languageCode {
```

Comment:
The and in this method name is nonstandard but still very readable. I would leave it but for future methods try using the standard of no and